src/Options/index.js

```diff
@@ -224,6 +218,10 @@ export interface ParseServerOptions {
   :ENV: PARSE_SERVER_PLAYGROUND_PATH
   :DEFAULT: /playground */
   playgroundPath: ?string;
+  /* Enables database hooks to update single schema cache. Set to true if using multiple Parse Servers instances connected to the same database.
+  :ENV: PARSE_SERVER_ENABLE_SCHEMA_HOOKS
+  :DEFAULT: false */
+  enableSchemaHooks: ?boolean;
```

Comment:
To sum it up from the comments, the only thing that I would change is to move the parameter into the `databaseOptions` instead of keeping it at root level, because it seems only DB related.

```
const api = new ParseServer({
  databaseOptions: {
    enableSchemaHooks: true
  },
});

```

And the previous `replicaSet` option has been removed, correct? So the developer basically need to make sure to only set `enableSchemaHooks: true` if they run a replica set.

Reply:
I removed replicaSet because using MongoDB ChangeStream requires more than just replicaSet. Also I need an option that works with Postgres. Postgres wouldn't need to have `enableSchemaHooks` I added it for the flaky tests.

I'll make the option change

Reply:
@mtrezza I moved `enableSchemaHooks` to `databaseOptions`